megatron/fused_kernels/__init__.py

```diff
@@ -12,13 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Copyright (c) 2022, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Why include this copyright?

Reply:
Unless this file is verbatim from NVIDIA's code, the copyread header should look more like [this](https://github.com/EleutherAI/gpt-neox/blob/9ddff09d360fa9e281c96df48ad1bf42c05eb5f1/megatron/training.py)

Reply:
This code is borrowed from 
https://github.com/microsoft/Megatron-DeepSpeed/blob/main/megatron/fused_kernels/__init__.py

I have pushed a correct format

Reply:
It's not quite the same wording as we use elsewhere, but the new header looks adequate.